.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Dispatch on Label
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
It might make sense to not restrict to the `master` branch, since that might change in the future, and custom branches on the repo might also want CI coverage. So

`on: [push, pull_request]`

should suffice

Reply:
@stoically Great callout. We will be updating the default branch name in the direct future.